test/View/Helper/FlashMessengerTest.php

```diff
@@ -376,6 +381,7 @@ public function testCanDisplayListOfCurrentMessagesCustomisedByConfigSeparator()
         $helperPluginManager = $services->get('ViewHelperManager');
         $helper              = $helperPluginManager->get('flashmessenger');
 
+        $helper->setView(new PhpRenderer());
```

Comment:
Resolves

```LaminasTest\Mvc\Plugin\FlashMessenger\View\Helper\FlashMessengerTest::testCanDisplayListOfCurrentMessagesCustomisedByConfigSeparator
TypeError: method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given

/usr/src/myapp/vendor/laminas/laminas-view/src/Helper/FlashMessenger.php:344
/usr/src/myapp/vendor/laminas/laminas-view/src/Helper/FlashMessenger.php:165
/usr/src/myapp/vendor/laminas/laminas-view/src/Helper/FlashMessenger.php:128
/usr/src/myapp/test/View/Helper/FlashMessengerTest.php:384
```

Please advise if another solution is preferred.

Reply:
@bfoosness
Please check #11

Reply:
I updated to reflect a similar change to https://github.com/laminas/laminas-view/blob/3.0.x/src/Helper/FlashMessenger.php#L344

Currently tests are failing when we use the lowest dependencies for PHP 8 (https://github.com/laminas/laminas-mvc-plugin-flashmessenger/pull/10/checks?check_run_id=2313255852) due to the same line above at version 2.10 of laminas-view: https://github.com/laminas/laminas-view/blob/2.10.0/src/Helper/FlashMessenger.php#L344

Reply:
Is the solution to bump the minimum version of laminas-view to 2.12, which is when it was fixed? https://github.com/laminas/laminas-view/blob/2.12.0/src/Helper/FlashMessenger.php#L344

Reply:
Why _does_ this class live in both this repository and laminas-view?

Reply:
BC reasons. It will be removed when next major is released

Reply:
Yes, bump the laminas-view but also open issue about FlashMessager helper from laminas-view being preferred over the one provided by this package. This is either a test setup issue or a legit problem that needs to be investigated for a potential fix.

Reply:
I thought as much. Unfortunately it looks like laminas-view 2.12 requires PHP 8, so by making that the minimum we'd be forcing PHP 8 to use this package?

Reply:
Never mind; it allows ^7.3, too.

Reply:
It looks like https://github.com/laminas/laminas-view/issues/16 already exists. Does that cover us here?